MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -303,7 +287,7 @@ store to msg.nmea.lat
       break;
 
     default:
-        if (offset < NMEA_BUF_LENGTH) 
+        if (offset < 15) // offset reset to 0 for every param so never be longer tnen 15
             string[offset++] = c;
```

Comment:
I found NMEA_BUF_LENGTH defined in protocols.h but do not know does it used anywhere else. Also, I try and experiment with different values without success.